tests/public-api/conftest.py

```diff
@@ -74,8 +75,8 @@ def simcore_docker_stack_and_registry_ready(
     simcore_services_ready: None,
 ) -> Dict:
     for attempt in Retrying(
-        wait=wait_fixed(5),
-        stop=stop_after_attempt(60),
+        wait=wait_random(2, 10),
+        stop=stop_after_delay(5 * _MINUTE),
```

Comment:
Just want to check if you sure about the `wait_random` and the 5 minute bump?

Reply:
at this point all services in the stack are running ( guaranteed by ``simcore_services_ready``). Here we check that the web-server API is responsive by requesting the health-check entrypoint (once more). 
These delay should actually be smaller and the wait could easily be fixed